<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShelvesConfigPage</class>
 <widget class="QWidget" name="ShelvesConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shelves Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="description_label">
     <property name="text">
      <string>The Shelves plugin organizes your music files by top-level folders (shelves) in your music library.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="shelves_group">
     <property name="title">
      <string>Known Shelves</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="shelves_label">
        <property name="text">
         <string>Manage your virtual shelves (top-level folders in your music library):</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="shelf_list">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="add_shelf_button">
          <property name="text">
           <string>Add Shelf...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_shelf_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove Selected</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="scan_shelves_button">
          <property name="toolTip">
           <string>Automatically detect shelves from your music library directory</string>
          </property>
          <property name="text">
           <string>Scan Music Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="workflow_group">
     <property name="title">
      <string>Workflow</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="workflow_enabled">
        <property name="text">
         <string>Use workflow</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="workflow_transitions">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Transition</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>from</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="workflow_stage_1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="workflow_stage_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="info_group">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Information</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="tag_info_label">
        <property name="text">
         <string>&lt;b&gt;Tag used:&lt;/b&gt; shelf</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="usage_info_label">
        <property name="text">
         <string>&lt;b&gt;Usage:&lt;/b&gt; Right-click on albums or tracks to set/change the shelf name. The plugin automatically detects shelves from file paths during scanning.</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="script_info_label">
        <property name="text">
         <string>&lt;b&gt;File naming script example:&lt;/b&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="script_preview">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>$set(_basefolder,$if(%shelf%,%shelf%))
$set(_basefolder,$if($eq(%_basefolder%,Incoming),Standard,%_basefolder%))
$set(_basefolder,$if($not($eq(%_basefolder%,)),%_basefolder%/))

%_basefolder%$if2(%albumartist%,%artist%)/%album%/%title%</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
